components/TextField/index.js

```diff
@@ -1,15 +1,36 @@
 import React from 'react';
 import MaterialTextField from '@material-ui/core/TextField';
+import PropTypes from 'prop-types';
 import { styled } from '@material-ui/core/styles';
 
 const StyledTextField = styled(MaterialTextField)({
   maxWidth: '500px',
   minWidth: '300px',
+  margin: '10px 0',
+  width: '100%',
 });
 
 
-const TextField = ({ ...others }) => (
-  <StyledTextField {...others} />
-);
+const TextField = ({
+  field, helperText, form, ...others
+}) => {
+  const fieldName = field && field.name;
```

Comment:
this might be a good use case for the new optional chaining method in JS. Docs [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)

So instead you could do: 
`const fieldName = field?.name`

Reply:
Yeah agreed, just need to configure Babel and eslint to support it.